packages/orbit-components/src/Stack/Stack.stories.tsx

```diff
@@ -190,7 +190,6 @@ export const TabletProperties = () => {
   const align = select("Align", Object.values(ALIGNS), ALIGNS.END);
   const justify = select("Justify", Object.values(JUSTIFY), undefined);
   const spacing = select("Spacing", Object.values(SPACINGS), undefined);
-  const spaceAfter = select("spaceAfter", Object.values(SPACINGS_AFTER), undefined);
```

Comment:
This is to stay, just like the other deletion. What was not to add was the prop to `<Stack>`. This file is to be left untouched in this PR